src/behaviors/subgrids.js

```diff
@@ -64,6 +64,7 @@ exports.mixin = {
         }, this);
 
         this.shapeChanged();
```

Comment:
this is used for all other behavior changed.
Added a new one here just for subgrid change event